replace G4SubString with G4String since former was removed in Geant4 10.04
see also https://github.com/OpenGATE/Gate/pull/162
author: Kenneth Hoste (HPC-UGent)
--- gate_v8.0/source/digits_hits/include/GateTokenizer.hh.orig	2017-12-14 12:57:13.630249979 +0100
+++ gate_v8.0/source/digits_hits/include/GateTokenizer.hh	2017-12-14 12:57:23.920311457 +0100
@@ -39,7 +39,7 @@
   static BrokenString BreakString(const G4String& stringToBreak,char separator);
 
 public:
-  G4SubString operator()(const char* str=" \t\n",size_t l=0)
+  G4String operator()(const char* str=" \t\n",size_t l=0)
     {
       return G4Tokenizer::operator()(str,l);
     }
